app/dataMappers/filmDataMapper.js

```diff
@@ -0,0 +1,35 @@
+const client = require('./dataBase');
+
+const dataMapper = {
+
+  async getdAllFilms() {
+    const query = {
+      text: 'SELECT * FROM film',
+    };
+    const result = await client.query(query);
+    return result.rows;
+  },
+
+  async getFilmsByDate(date) {
+    const startDate = (date - 1);
+    const endDate = (date + 10);
```

Comment:
[Clean architecture] C'est pas le rôle du dataMapper de faire ce genre logique business !

A choisir, j'aurais donc fait une fonction plutôt comme ça : 

```suggestion
  async getFilmsReleasedBetween(startReleaseYear, endReleaseYear) {
    // ...
```

Comme ça on ne trompe personne au niveau de l'appel au controlleur ! Car j'ai cru qu'on aurait uniquement les films de l'année demandé, ni plus ni moins. Et en fait c'est pas le cas !